doc/resources.md

```diff
@@ -41,3 +41,5 @@ See [this page](./github.md) for more information.
 - [YouTube](https://www.youtube.com/@NixOS-Foundation)
 - [Twitter](https://twitter.com/nixos_org)
 - [Mastodon](https://chaos.social/@nixos_org)
+- [LinkedIn](https://www.linkedin.com/company/nixos-foundation)
+- [Bluesky](https://bsky.app/profile/nixos-org.bsky.social)
```

Comment:
Ideally we also keep track of who has access to these channels. While I didn't spend the time to figure that out for the above resources, we should definitely document it for new ones:

```suggestion
- [LinkedIn](https://www.linkedin.com/company/nixos-foundation)
  - Access: ???
- [Bluesky](https://bsky.app/profile/nixos-org.bsky.social)
  - Access: ???
```

Note that this also means the docs here should be updated whenever it changes :)

Reply:
LinkedIn now done in https://github.com/NixOS/org/pull/153